src/models/multiChart.js

```diff
@@ -90,7 +90,7 @@ nv.models.multiChart = function() {
                     })
                 });
 
-            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return d.x } ))
+            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return chart.x()(d) } ))
```

Comment:
Oh, it looks like right above this it uses the getX function (which is what chart.x() actually returns)... so should probably just use that for consistency...     sorry!   

If you change that (and please verify it actually works for your use case) then I'll merge it in :)